build.gradle

```diff
@@ -1,22 +1,35 @@
 // Required by spotless. https://github.com/diffplug/spotless/issues/747
 buildscript {
+    ext {
+        // plugin versions
+        vSpringBoot = '3.3.4'
+        vSpringDepMgt = '1.1.6'
+        vHidetakeSwagger = '2.19.2'
+        vGoogleProtobuf = '0.9.4'
+
+        // library versions
+        vSpringWeb = '6.1.13'
+        vPmd = '7.1.0'
+        vTerraCommonLib = '1.1.16-SNAPSHOT'
+        vApacheCommonsText = '1.12.0'
+        vSlf4jApi = '2.0.16'
+        vSpotBugs = '4.8.5'
+        vJackson = '2.17.2'
+        vJersey = '3.1.8' // Java 17 compatible
+        vSwaggerAnnotations = '2.2.22'
+        vSwaggerCodegen = '3.0.57'
+        vPicocliVersion = '4.7.6'
+        vJunit5 = '5.10.2'
+    }
     repositories {
         mavenCentral()
     }
 }
 
-gradle.ext {
-    vSpringBoot = '3.3.3'
-    vSpringWeb = '6.1.13'
-    vPmd = '7.1.0'
-    vTerraCommonLib = '1.1.16-SNAPSHOT'
-    vApacheCommonsText = '1.12.0'
-    vSlf4jApi = '2.0.16'
-    vSpotBugs = '4.8.5'
-    vJackson = '2.17.2'
-    vJersey = '3.1.8' // Java 17 compatible
-    vSwaggerAnnotations = '2.2.22'
-    vSwaggerCodegen = '3.0.57'
-    vPicocliVersion = '4.7.6'
-    vJunit5 = '5.10.2'
+tasks.named('dependencies') {
```

Comment:
This configures a root task dependencies that depends on below subtasks. hence auto triggered.